backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -136,17 +156,17 @@ public UserDto findUserByEmail(String email) {
   }
 
   @Override
-  public List<UserDto> listAllUsers() {
+  public List<UserResponse> listAllUsers() {
     List<UserEntity> entityList = userRepository.findAll();
     return entityList.stream()
-        .map(e -> modelMapper.map(e, UserDto.class))
+        .map(e -> modelMapper.map(e, UserResponse.class))
         .collect(Collectors.toList());
   }
 
   @Override
   public String confirmEmail(String token) {
-    emailUtils.validateJwtToken(token);
-    String email = emailUtils.getEmailFromJwtEmailToken(token);
+    JwtUtils.validateEmailToken(token, JwtUtils.EMAIL_VERIFICATION_PURPOSE);
```

Comment:
Since you are refactoring this and this both methods are doing pretty much the same thing, can you merge them into one method that will validate the token and return the subject or whatever is more suitable?

Reply:
Of course, the second method is 1 line anyways, makes more sense if that's just the return value.